main.go

```diff
@@ -1,32 +1,37 @@
 package main
 
 import (
+	_ "embed"
 	"os"
+	"strings"
 
 	"github.com/gookit/color"
-	"github.com/joho/godotenv"
-
-	"github.com/skuid/tides/cmd"
-	"github.com/skuid/tides/pkg/logging"
+	"github.com/skuid/skuid-cli/cmd"
+	"github.com/skuid/skuid-cli/pkg"
+	"github.com/skuid/skuid-cli/pkg/logging"
 )
 
+// VERSION_NAME has to be here instead of in the constants package
+// because embed does not allow relative paths (parent directories)
+
+//go:embed .version
+var VERSION_NAME string
+
+func init() {
+	VERSION_NAME = strings.TrimSpace(VERSION_NAME)
```

Comment:
skuid-cli is not sending the correct User-Agent, looks like init is happening after the initialization of SkuidUserAgent variable. I compiled it using `make docker-build` so it is injecting the version value from the file.

![image](https://user-images.githubusercontent.com/30167939/205377856-fe1d1981-60b5-4139-9b71-fdbbcd428027.png)

![image](https://user-images.githubusercontent.com/30167939/205378533-2d7490c3-d354-49be-8731-398b35d15982.png)

Reply:
![image](https://user-images.githubusercontent.com/30167939/212183944-aa1a2064-0030-4e8c-a796-33b8f145042a.png)
I'm still not getting the correct version in the header, what I did is make docker-build which supposes that is embedding the version in the binary.